Sources/ObjCSupport/LOKButtonLayout.swift

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018 LinkedIn Corp.
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import UIKit
+
+@objc public class LOKButtonLayout: LOKBaseLayout {
```

Comment:
I am wondering why we can't directly create objective-c classes for wrappers? With objective-c wrappers, it might be much faster with generating swift interfaces. Also, we can separate them to different pod, so the `LayoutKit` is purely swift `LayoutKit` without objc support.

Reply:
Swift was chosen for these wrappers since this codebase already uses Swift and for simplicity I'd like to keep it single-language. I doubt there would much difference in speed, either at editing, build, or run time.

Regarding Pods, I'm planning to exclude these wrappers from the current Pod and include them in a new Pod.